controller/controlplane/manager_options.go

```diff
@@ -523,5 +523,9 @@ func WithTranslationOptions(opts *operatorv2alpha1.ControlPlaneTranslationOption
 			c.CombinedServicesFromDifferentHTTPRoutes = *opts.CombinedServicesFromDifferentHTTPRoutes ==
 				operatorv2alpha1.ControlPlaneCombinedServicesFromDifferentHTTPRoutesStateEnabled
 		}
+		if fc := opts.FallbackConfiguration; fc != nil && fc.UseLastValidConfig != nil {
+			c.UseLastValidConfigForFallback = *fc.UseLastValidConfig ==
+				operatorv2alpha1.ControlPlaneFallbackConfigurationStateEnabled
```

Comment:
Could we use 
 ```
c.UseLastValidConfigForFallback =
 (*fc.UseLastValidConfig == operatorv2alpha1.ControlPlaneFallbackConfigurationStateEnabled)
```
to make it clearer? It is a little confusing when the `==` expression is separated into 2 lines.